lib/api/utilities/text_renderer.rb

```diff
@@ -39,7 +39,11 @@ def initialize(text, format: nil, object: nil)
         @text = text
         @format = format
         @object = object
-        @project = object.project if object.respond_to?(:project)
+        if object.respond_to?(:project)
+          @project = object.project
+        else
+          @project = object
```

Comment:
I can't remember why I wanted to have an else case here at all o0. I will remove it and implement your suggestion.